docs/pages/api/routes.md

```diff
@@ -38,7 +36,69 @@ type OutputScene = {
 
 ***
 
-<Docs />
+### RegisterInputRequest
```

Comment:
Why can't we use generated docs here?

Reply:
I explained the problems in the PR description.

We want to use generated docs here, but docs generation needs to be adjusted to produce sth that will fit otherwise it looks very bad.